implementationGuide/Mappings.md

```diff
@@ -36,14 +37,14 @@ In FHIR, the contact information is included in the *Organization* resource, whi
 |address | contact:*contact*.address.line |
 |zip | contact:*contact*.address.postalCode |
 |city | contact:*contact*.address.city |
-|country | contact:*contact*.address.county |
+|country | contact:*contact*.address.country |
 
 ### Collection
-### Network
-
-
 
 
+### Network
+This project does not contain a specific profile for biobank networks. While such networks could be represented as *Organization* resources, it is outside the scope for this project.
```

Comment:
I think it would be nice to link the resources here to there FHIR definition pages.

Reply:
Thanks for the suggestion, I additionally added a link to the BBMRI Directory.